packages/gatsby-cli/src/create-cli.ts

```diff
@@ -400,15 +401,21 @@ function buildLocalCommands(cli: yargs.Argv, isLocalSite: boolean): void {
     builder: yargs =>
       yargs
         .positional(`cmd`, {
-          choices: [`docs`],
-          describe: "Valid commands include `docs`.",
+          choices: [`docs`, `ls`],
```

Comment:
Not needed now, but I wonder if we should add `list` as an alias for `ls`. Windows people might not know what `ls` means

Reply:
Fair enough and easy to do. Let’s get this in and we can add it in the next pass.